game.lua

```diff
@@ -379,9 +379,9 @@ function displayTips(target)
 end
 
 function displayProfile(name, target)
-  local p = players[name] or players[name .. "#0000"]
+  local p = players[upper(name)] or players[upper(name) .. "#0000"]
   if p then
-    ui.addTextArea(900, closeButton .. "<p align='center'><font size='15'><b><BV>" .. p:getName() .."</BV></b></font></p><br><b>Level:</b> " .. tostring(p:getLevel()) .. "<BL><font size='12'> [" .. tostring(p:getXP()) .. "XP / " .. tostring(calculateXP(p:getLevel() + 1)) .. "XP]</font></BL><br><b>Money:</b> $" .. formatNumber(p:getMoney()) .. "<br><br><b>Working as a</b> " .. p:getJob() , target, 300, 100, 200, 130, nil, nil, 1, true)
+    ui.addTextArea(900, closeButton .. "<p align='center'><font size='15'><b><BV>" .. p:getName() .."</BV></b></font></p><br><b>Level:</b> " .. tostring(p:getLevel()) .. "<BL><font size='12'> [" .. tostring(p:getXP()) .. "XP / " .. tostring(calculateXP(p:getLevel() + 1)) .. "XP]</font></BL><br><b>Money:</b> $" .. formatNumber(float(p:getMoney())) .. "<br><br><b>Working as a</b> " .. p:getJob() , target, 300, 100, 200, 130, nil, nil, 1, true)
```

Comment:
I forgot to put the second argument for the float function and I'm on phone. Could you please put it as 1? Thank you.

Reply:
It'd be better if you could make that change because I don't have access to the fork and I'm not sure that I'd make the change after the merge